com.huma.browser.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Version=100.0a15
+Name=Hüma Browser
+Comment=Web browser by Hüma Project
+Exec=flatpak run com.huma.browser
```

Comment:
definitely NEVER a flatpak command here. It should be just the name of the executable.